pkg/controllers/managementuser/nodesyncer/nodessyncer.go

```diff
@@ -181,7 +181,10 @@ func (n *nodeSyncer) sync(key string, node *corev1.Node) (runtime.Object, error)
 		return nil, err
 	} else if ok {
 		node = node.DeepCopy()
-		node.Labels[UpgradeEnabledLabel] = "true"
+		// only linux nodes are supported in imported clusters
+		if node.Labels["kubernetes.io/os"] != "windows" {
+			node.Labels[UpgradeEnabledLabel] = "true"
```

Comment:
Do we need to reset the label to false for existing Windows nodes?

Reply:
I think so; I'll update the code.